taxi/views.py

```diff
@@ -36,6 +45,27 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(
+            self,
+            *,
+            object_list: list = None,
```

Comment:
add additional typehint for list inner objects like list[ANY_TYPE]
use typehint like: <ANY_TYPE> | None = None